lib/tasks/thor/setup.rb

```diff
@@ -83,6 +83,7 @@ def welcome
         default_user_id: 1
       )
 
+      User.create(email: 'adama@dradisframework.com')
```

Comment:
This seems to belong before the previous call, so L83 can be replaced with the actual `user.id` which probably will be 1 all the time anyway, but it seems a bit weird to invert the order...